src/planner/linux.rs

```diff
@@ -39,6 +39,10 @@ impl Planner for Linux {
             return Err(PlannerError::NixOs);
         }
 
+        if std::env::var("WSL_DISTRO_NAME").is_ok() && !Path::new("/run/WSL").exists() {
```

Comment:
Bad news: I was lied to that "`/run/WSL` existing means WSL2". Or at least, it doesn't determine if the _current_ WSL instance is, just that there is _some_ WSL2 instance available (I have both v1 and v2 instances, and /run/WSL exists in both).

But apparently only WSL2 has the `WSL_INTEROP` env var, so let's try that.